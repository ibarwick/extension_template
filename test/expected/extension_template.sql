/*
 * Regression test for extension_template
 */

SELECT * FROM function_template();
