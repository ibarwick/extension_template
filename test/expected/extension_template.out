/*
 * Regression test for extension_template
 */
SELECT * FROM function_template();
 function_template 
-------------------
 
(1 row)

